packages/nft-playbook/src/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "GAS_LIMIT" : "200000",
+    "server_uri" : "http://127.0.0.1:7545",
+    "priv_key_contract_owner" : "c13ab69ac6895785d03c6576f1044750aa0e505a7639972906ee466616fed958",
+    "priv_key_NFT_transmitter" : "c13ab69ac6895785d03c6576f1044750aa0e505a7639972906ee466616fed958",
```

Comment:
Secets like private keys should be retrieved from CLI / User and temporarily stored in memory, not in our config
=> this is fine for this current sprint but needs to be addressed later on

Reply:
i think JSON format is perfect for the config file. This is good for this sprint in my opinion.

Reply:
I agree in @phildree. All secrets should be read from commandline, not from static files.

Reply:
this will definitely change in the future. It's only here right now for testing purposes on testing blockchains only. Just like you mentioned.